language/move-lang/functional-tests/tests/libra/payment/cant_copy_resource.move

```diff
@@ -0,0 +1,11 @@
+script {
+use 0x1::LBR::LBR;
+use 0x1::Libra;
+
+fun main(account: &signer) {
+    let coin = Libra::zero<LBR>();
+    Libra::destroy_zero<LBR>(copy coin);
+    Libra::destroy_zero<LBR>(move coin);
+}
+}
+// check: MoveSourceCompilerError Invalid copy of owned resource value
```

Comment:
This test will will be removed, the same thing is tested in `move_check/translated_ir_tests/move/signer/is_resource_copy_loc.move`